### DISCUSSION TITLE
Hiding a PR review comment

### DISCUSSION THREAD

**Discussion by RokLenarcic** [community/community#65914] [Apps, API and Webhooks] [answered]
### Select Topic Area

Question

### Body

In PR Conversation tab you can see Review comments, and for each comment you can access a context menu at top right at `...`. In that context menu you can select `Hide`. And then you can specify a reason and it hides the comment.

Is there a way to do this action via API?

I am trying to remove an old review, and I can dismiss approving or rejecting reviews, but I cannot really remove "COMMENT" type reviews.

**Comment by arafajs** [comment, marked as answer]
Unfortunately there is no direct API to hide or remove review comments in GitHub.

**Reply by mskonovalov** [reply]
Awesome, is it possible to add it to the roadmap? Thanks

**Reply by lmmx** [reply]
Related https://github.com/orgs/community/discussions/19865
